nearby.py

```diff
@@ -5,67 +5,70 @@
 # this script needs python3.6 or newer because of the "f-string" syntax,
 # aka PEP 498's literal string interpolation. you'll also need geopy and requests:
 # pip3 install geopy requests
-
-from geopy.distance import geodesic as geo
+import datetime
 import re
 import sys
+
 import requests
-import datetime
+from geopy.distance import geodesic as geo
+
 
 # helper funcs
 def loc(d, prefix=''):
   return (d[f'{prefix}lat'], d[f'{prefix}lng'])
+
+
 def normalize_name(n):
   return re.sub(r'\W', '-', n.lower())
 
-homeid = None
-if len(sys.argv) > 1:
-  homeid = normalize_name(' '.join(sys.argv[1:]))
-else:
-  print("give your hotspot id (one-two-three) as the argument. spaces or dashes are fine.")
-
-print(f"normalized name: {homeid}")
-
-home = None
-hotspot = {}
-r = requests.get('https://network.helium.com/fetchHotspots')
-for h in r.json():
-  # save them by name
-  stdname = normalize_name(h['name'])
-  hotspot[stdname] = h
-  if stdname == homeid:
-    home = h
-
-if not home:
-  print(f"eep, we didn't find your hotspot name ({homeid}).")
-  sys.exit(-1)
-
-hotspots = []
-
-# now compare them to see if they're local
-print(f"dist score name")
-for h_id,h in hotspot.items():
-  dist = geo( loc(home), loc(h) ).miles
-  if dist < 30:
-    score = int(h['score']*100)
-    coord = f"{h['lat']:3.4f}, {h['lng']:3.4f}"
-    name = h['name']
-    result = (dist, score, name, coord)
-    hotspots.append(result)
-
-for result in reversed(sorted(hotspots, key=lambda x: x[0])):
-  print(f"{result[0]:4.1f} {result[1]:5} {result[2]:30} {result[3]}")
-
-# now pull peeps who have witnessed us: https://tedder.me/lols/witness-me-2.gif
-witret = requests.get(f'https://explorer.helium.foundation/api/witnesses/{home["address"]}')
-data = witret.json().get('data')
-if len(data):
-  print(f"\nhotspot name                   count    rssi  witness time")
-  for h in sorted(data, key=lambda x: x.get('recent_time')):
-    # h = dict of pocs that saw us
-    hist = h['hist']
-    rssi = max(hist, key=hist.get)
-    count = sum(hist.values())
-    print(f"{h['name']:25} {count:10}   {rssi:5}  {datetime.datetime.fromtimestamp(h['recent_time']/1000**3).isoformat(timespec='minutes')}")
 
+if __name__ == '__main__':
```

Comment:
to save me the indentation work, can you just change this to a monolithic function and move the `__main__` to the bottom? It doesn't have to be perfectly architected, just a basic call.

(if that doesn't make sense I will do it after the merge)

Reply:
Heh. I almost did that, but have run across folks who didn't like a `def main:` Doing that now.

Reply:
I think that's what you were asking for. If not, feel free to modify however you want